Marlin/Configuration_adv.h

```diff
@@ -1554,7 +1554,10 @@
    * Endstops must be activated for this option to work.
    */
   //#define SD_ABORT_ON_ENDSTOP_HIT
-
+  #if ENABLED(SD_ABORT_ON_ENDSTOP_HIT)
+    //#define SD_ABORT_ON_ENDSTOP_HIT_CUSTOM_GCODE "G28XY" //G-code to run on SD Abort Print (e.g., "G28XY" or "G27")
```

Comment:
I forgot to change the comment in my last reply. Sorry about that:

`// G-code to run on endstop hit (e.g., "G28XY" or "G27")`

Reply:
Oh, i didn't see that either. Fixing...